### DISCUSSION TITLE
Media Player Volume for Dials?

### DISCUSSION THREAD

**Discussion by ct20019** [cgiesche/streamdeck-homeassistant#272] [Help and Support] [answered]
Hello,

Is it possible to control the volume of a media_player with the dials? I have seen several posts on how to achieve changing the brightness of lights, so I figured changing the volume level of a media device might be possible. Below is what i have attempted to add, I'm sure i am not adding it in correctly.

![Stream Deck 1](https://github.com/cgiesche/streamdeck-homeassistant/assets/107083912/e790f166-681b-4128-9466-447469c77a23)
![Stream Deck 2](https://github.com/cgiesche/streamdeck-homeassistant/assets/107083912/c4be6acb-50df-461d-bd34-08c03e90c4ea)

**Comment by artistro08** [comment, marked as answer]
I just figured this out, Turns out you can use math in those curly brackets

```json
{ "volume_level": {{ rotationPercent / 100 }} }
```

**Reply by ct20019** [reply]
This worked! Thank you for finding the solution for this.

**Reply by artistro08** [reply]
Just an update here's a better version of the above: https://github.com/cgiesche/streamdeck-homeassistant/discussions/275
